consumer/src/main/kotlin/streams/procedures/StreamsSinkProcedures.kt

```diff
@@ -27,13 +39,31 @@ class StreamsSinkProcedures {
         }
 
         val properties = config?.mapValues { it.value.toString() } ?: emptyMap()
-        val configuration = StreamsSinkProcedures.streamsEventSinkConfigMapper.convert(config = properties)
+        val configuration = streamsEventSinkConfigMapper.convert(config = properties)
 
         val data = readData(topic, config ?: emptyMap(), configuration)
 
         return data.map { StreamResult(mapOf("data" to it)) }.stream()
     }
 
+    @Procedure("streams.sink.start")
```

Comment:
This should return a boolean of whether or not the operation worked, rather than returning nothing.

Reply:
Additionally, what will happen if this is called when no streams sink configuration is specified?  (Exception should be the result)

Reply:
@moxious I'll add the exception in case is not a LEADER or a single instance configuration.
About the config, we'll reuse the same configuration provided at startup time, so no change can be provided, we can work on a new PR where the user can provide the new config via the procedure wdyt?

Reply:
All good -- except defer the "provide new config via procedure".  This is a big complicated point for all plugins, how to support dynamic config.  It'll cause separate problems if the config at runtime doesnt' match what's in the config file.  I don't want to get into that with the functionality here until we have a better solution for dynamic config of plugins, which like we discussed before isn't really a neo4j-streams issue.    If we try to do dynamic config without the conf file, it's going to get really complicated quickly with feature requests & bugs, and we need a better overall approach for that rather than adding as an incremental feature.